.github/workflows/oss_latest_snapshot_push.yml

```diff
@@ -41,6 +44,7 @@ jobs:
           docker buildx build --load \
             --build-arg HZ_VERSION=${{ github.event.inputs.HZ_VERSION }} \
             --build-arg HZ_VARIANT=${{ matrix.variant }} \
+            --build-arg JDK_VERSION=${{ matrix.jdk }} \
```

Comment:
Use jdk variable in `Store docker logs as artifact` and `Get docker logs`

Reply:
done